config/settings.py

```diff
@@ -46,6 +46,7 @@
     "django_countries",
     "tailwind",
     "base",
+    "heya",
```

Comment:
@coderabbitai The migration has already been tested in is working fine. All migrations will eventually be reset in a future PR since no data exists yet